<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setting</class>
 <widget class="QMainWindow" name="setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>setting</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="developer_mode"/>
        </item>
        <item>
         <widget class="QLineEdit" name="main_server"/>
        </item>
        <item>
         <widget class="QLineEdit" name="main_server_port"/>
        </item>
        <item>
         <widget class="QLineEdit" name="steam_and_water_screen"/>
        </item>
        <item>
         <widget class="QLineEdit" name="steam_and_water_screen_port"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_14"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_10"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_16"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="developer_mode_value"/>
        </item>
        <item>
         <widget class="QLineEdit" name="main_server_value"/>
        </item>
        <item>
         <widget class="QLineEdit" name="main_server_port_value"/>
        </item>
        <item>
         <widget class="QLineEdit" name="steam_and_water_screen_value"/>
        </item>
        <item>
         <widget class="QLineEdit" name="steam_and_water_screen_port_value"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_13"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_11"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_15"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="apply_button">
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>429</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
